share/applications/kakoune-connect.desktop

```diff
@@ -2,4 +2,4 @@
 Type=Application
 Name=Kakoune (C)
 Exec=kak-connect %F
-Icon=kakoune-icon.png
+Icon=kakoune.svg
```

Comment:
Shouldn’t be `kakoune` without the extension?

Reply:
The documentation doesn't suggest that. I think it is correct to put it with extension.

Reply:
Can we resolve this?